scripts/Minecraft.zs

```diff
@@ -2398,6 +2403,12 @@ recipes.addShaped(<minecraft:fishing_rod>, [
 [null, <ore:stickLongWood>, <minecraft:string>],
 [<ore:stickLongWood>, <ore:craftingToolWireCutter>, <ore:ringAnyIron>]]);
 
+// --- Paper
+recipes.addShaped(<minecraft:paper> * 2, [
+[null, <minecraft:stone_slab>.reuse(), null],
+[<ore:dustPaper>, <ore:dustPaper>, <ore:dustPaper>],
+[null, <minecraft:stone_slab>.reuse(), null]]);
+
```

Comment:
Put this change in gt code please

Reply:
@Dream-Master , you mean a gt.zs ?

Reply:
As part of https://github.com/GTNewHorizons/GT5-Unofficial

Reply:
@Prometheus0000, I understood that code GT, but where exactly?

Reply:
I would be glad if you can add it 
or tell me where to add it 😽